v03_pipeline/lib/tasks/update_lookup_table.py

```diff
@@ -73,51 +47,34 @@ def complete(self) -> bool:
 
     def requires(self) -> list[luigi.Task]:
         return [
-            WriteRemappedAndSubsettedCallsetTask(
-                self.reference_genome,
-                self.dataset_type,
-                self.sample_type,
-                callset_path,
-                project_guid,
-                project_remap_path,
-                project_pedigree_path,
-                imputed_sex_path,
-                self.ignore_missing_samples_when_remapping,
-                self.validate,
-                False,
+            self.clone(
+                WriteRemappedAndSubsettedCallsetTask,
+                project_guid=project_guid,
+                project_remap_path=project_remap_path,
+                project_pedigree_path=project_pedigree_path,
+                force=False,
             )
             for (
-                callset_path,
                 project_guid,
                 project_remap_path,
                 project_pedigree_path,
-                imputed_sex_path,
-            ) in callset_project_pairs(
-                self.callset_paths,
+            ) in zip(
                 self.project_guids,
                 self.project_remap_paths,
                 self.project_pedigree_paths,
-                self.imputed_sex_paths,
+                strict=False,
```

Comment:
`strict=False` [makes](https://docs.python.org/3/library/functions.html#zip) it so that if any of the 3 lists are of different length we'll never know - do we expect these to ever be of different lengths?

Reply:
yeah! I think this is worth changing.  `ruff` complains if you `zip` without the param... and I do think it's impossible for the lists to not be the same length with the way it's been simplified now....but throwing an error is definitely clearer!